Cargo.toml

```diff
@@ -39,6 +39,8 @@ toml = "0.7.3"
 walkdir = "2.3"
 filetime = "0.2.9"
 itertools = "0.12"
+pulldown-cmark = "0.11"
+rinja = { version = "0.3", default-features = false, features = ["config"] }
```

Comment:
You might have heard of `askama`. I contributed to it a lot but some issues happened there (nothing bad, just different opinion on some specific things) and another `askama` maintainer and I decided to fork it.

You can see more information about it [here](https://blog.guillaume-gomez.fr/articles/2024-07-31+docs.rs+switching+jinja+template+framework+from+tera+to+rinja).

The biggest advantage of `rinja` is that if the template code is wrong, it will prevent compilation (which is the big reason why we moved to this framework for docs.rs).